libyul/optimiser/ControlFlowSimplifier.cpp

```diff
@@ -60,8 +59,9 @@ void removeEmptyDefaultFromSwitch(Switch& _switchStmt)
 
 void removeEmptyCasesFromSwitch(Switch& _switchStmt)
 {
-	bool hasDefault = boost::algorithm::any_of(
-		_switchStmt.cases,
+	bool hasDefault = std::any_of(
+		_switchStmt.cases.begin(),
```

Comment:
:(

Reply:
You didn't need to merge it, it was up for debate.